anchor/views/posts/add.php

```diff
@@ -1,5 +1,15 @@
 <?php echo $header; ?>
 
+<?php
```

Comment:
Loading the `$meta` needs to be done in the routes and not the view. Otherwise this looks great! Once this has been resolved it can be merged in :)